src/locale/_build/src/components/contact/Contact.js.json

```diff
@@ -0,0 +1,18 @@
+{
```

Comment:
I'm wondering if we need the `_build` directory in git? I'm not really sure, can you investigate?